### DISCUSSION TITLE
Quality of Life  - Has anyone built/use something akin to a “add to exception” button for the Alerts screen? (v2.3)

### DISCUSSION THREAD

**Discussion by TFWol** [Security-Onion-Solutions/securityonion#7879] [Unsupported Versions] [answered]
Pretty much what the title says.
I self-host a lot of stuff and like to reduce time sinks as I tend to go down rabbit holes.

Heck, even a Userscript would be useful.
It can be discouraging needing to edit files all the time when you just want to get on with your day.

I would like to look at the alerts that catch my eye and go ‘these are false and... _*click*_’ then continue what I was doing, browsing YouTube, etc.

Would be super interested in what anyone has.

**Comment by dougburks** [comment, marked as answer]
> Heck, even a Userscript would be useful.
> It can be discouraging needing to edit files all the time when you just want to get on with your day.

Depending on what you're trying to do, `so-rule` might be able to help you avoid editing files:
https://docs.securityonion.net/en/2.3/managing-alerts.html#so-rule

We have plans to add this kind of functionality into Security Onion Console.

**Reply by TFWol** [reply]
Thanks for the reply.
Yeah, I had followed https://www.youtube.com/watch?v=1jEkFIEUCuI to tune some of the rules a few weeks ago, very useful.

> We have plans to add this kind of functionality into Security Onion Console.

Glad to see it's a consideration; I'm a bit burned out with scripting.
